e2e-tests/spring-rest-mongo/src/test/java/org/evomaster/e2etests/spring/rest/mongo/foo/ExtraFitnessMongoTest.java

```diff
@@ -0,0 +1,271 @@
+package org.evomaster.e2etests.spring.rest.mongo.foo;
+
+import com.foo.mongo.MyMongoAppEmbeddedController;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import com.google.inject.TypeLiteral;
+import org.evomaster.client.java.controller.internal.db.StandardOutputTracker;
+import org.evomaster.core.Main;
+import org.evomaster.core.database.DbAction;
+import org.evomaster.core.problem.rest.*;
+import org.evomaster.core.problem.rest.auth.NoAuth;
+import org.evomaster.core.problem.rest.param.Param;
+import org.evomaster.core.search.EvaluatedIndividual;
+import org.evomaster.core.search.FitnessValue;
+import org.evomaster.core.search.service.FitnessFunction;
+import org.evomaster.e2etests.spring.rest.mongo.SpringRestMongoTestBase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ExtraFitnessMongoTest extends SpringRestMongoTestBase {
+
+    private static final MyMongoAppEmbeddedController sutController = new MyMongoAppEmbeddedController();
+
+    @BeforeAll
+    public static void init() throws Exception {
+        SpringRestMongoTestBase.initClass(sutController);
+    }
+
+    @BeforeEach
+    public void turnOnTracker() {
+        StandardOutputTracker.setTracker(true, sutController);
+
+    }
+
+    @AfterEach
+    public void turnOffTracker() {
+        StandardOutputTracker.setTracker(false, sutController);
+    }
+
+    @Test
+    public void testExtraFitnessNoDocumentsFound() {
+
+        String[] args = new String[]{
```

Comment:
see `RestTestBase.getArgsWithCompilation`
Can have default args in `SpringRestMongoTestBase`, and then add the extra ones you need in each test

Reply:
ok